tests_e2e/test_app.py

```diff
@@ -0,0 +1,58 @@
+import os
+import pytest
+import requests
+import time
+from dotenv import find_dotenv, load_dotenv
+from selenium import webdriver
+from threading import Thread
+from todo_app.app import create_app
+from todo_app.trello_api import create_board
+
+board_id = ''
+
+def create_trello_board():
+    url = 'https://api.trello.com/1/boards/'
+    query = { 'key': os.environ.get('TRELLO_API_KEY'), 'token': os.environ.get('TRELLO_API_TOKEN'), 'name': os.environ.get('BOARD_NAME') }
+    response = requests.post(url, params=query)
+    result = response.json()
+    return result['id']
+
+def delete_trello_board():
+    url = f'https://api.trello.com/1/boards/{board_id}'
+    query = { 'key': os.environ.get('TRELLO_API_KEY'), 'token': os.environ.get('TRELLO_API_TOKEN') }
+    requests.delete(url, params=query)
+
+@pytest.fixture(scope='module')
+def app_with_temp_board():
+    file_path = find_dotenv('.env')
+    load_dotenv(file_path, override=False)
+    os.environ['BOARD_NAME'] = 'Test Board'
+    global board_id
```

Comment:
No need for this `board_id` to be a global variable (and generally better to avoid global variables). Just pass it as a parameter to `delete_trello_board`